simtools/corsika/corsika_histograms.py

```diff
@@ -75,14 +75,6 @@ def __init__(self, input_file, label=None, output_path=None, hdf5_file_name=None
         else:
             self.hdf5_file_name = hdf5_file_name
 
-        self._initialize_attributes()
-        self.read_event_information()
-        self._initialize_header()
-
-    def _initialize_attributes(self):
-        """
-        Initializes the class attributes.
-        """
         self._telescope_indices = None
         self._telescope_positions = None
         self.num_events = None
```

Comment:
lines 78 to 110 should be maintained.

These changes here are breaking the test. I suggest returning it to the previous form and change only upper case D to lower case d.

Reply:
Nothing has been removed, these functions are called (as before) after the init. 

The three functions are called after all variables have been initialized (class variable init should take place in the `__init__` function, this is why I have changed it.

Reply:
I dont mind putting everything in one function if the fuction is not used elsewhere. but the order seems to be important